controllers/v1beta1/build_deletionhandlers.go

```diff
@@ -213,23 +213,29 @@ Build cancelled
 		}
 		// send any messages to lagoon message queues
 		// update the deployment with the status
-		r.cancelledBuildStatusLogsToLagoonLogs(ctx, opLog, &lagoonBuild, &lagoonEnv)
-		r.updateCancelledDeploymentAndEnvironmentTask(ctx, opLog, &lagoonBuild, &lagoonEnv)
-		r.cancelledBuildLogsToLagoonLogs(ctx, opLog, &lagoonBuild, allContainerLogs)
+		r.buildStatusLogsToLagoonLogs(ctx, opLog, &lagoonBuild, &lagoonEnv, true)
+		r.updateDeploymentAndEnvironmentTask(ctx, opLog, &lagoonBuild, &lagoonEnv, true)
+		r.buildLogsToLagoonLogs(ctx, opLog, &lagoonBuild, allContainerLogs, true)
 	}
 	return nil
 }
 
-// cancelledBuildLogsToLagoonLogs sends the build logs to the lagoon-logs message queue
+// buildLogsToLagoonLogs sends the build logs to the lagoon-logs message queue
 // it contains the actual pod log output that is sent to elasticsearch, it is what eventually is displayed in the UI
-func (r *LagoonBuildReconciler) cancelledBuildLogsToLagoonLogs(ctx context.Context,
+func (r *LagoonBuildReconciler) buildLogsToLagoonLogs(ctx context.Context,
 	opLog logr.Logger,
 	lagoonBuild *lagoonv1beta1.LagoonBuild,
 	logs []byte,
+	cancelled bool,
```

Comment:
Looking at the code, I wonder if there's not an argument to be made to turn this into an enum or something representing the buildstate - this may make it clearer when reading the client code what's actually happening when we're calling the function?

Reply:
Ahh yes, good idea. This will be easy to do, already have state representation